pages/Announcements/Overview.md

```diff
@@ -20,14 +20,14 @@ All Announcements have a [signature](/Announcements/Signatures) to validate the
 
 Each Announcement has a enumerated type for use when separating out a stream of Announcements.
 
-| Value | Name | Description | DSNP Announcement URI |
-|------ | ---- | ----------- | -------------------- |
-| 0 | Reserved | reserved for future use | - |
-| 1 | [Graph Change](/Announcements/Types/GraphChange) | social graph changes | no |
-| 2 | [Broadcast](/Announcements/Types/Broadcast) | a public post | YES |
-| 3 | [Reply](/Announcements/Types/Reply) | a public response to a Broadcast | YES |
-| 4 | [Reaction](/Announcements/Types/Reaction) | a public visual reply to a Broadcast | no |
-| 5 | [Profile](/Announcements/Types/Profile) | a profile | YES |
+| Value | Name | Description | DSNP Announcement URI | Tombstone Allowed |
+|------ | ---- | ----------- | --------------------- | ----------------- |
+| 0 | [Tombstone](/Announcements/Types/Tombstone) | an invalidation of another announcement | no | no |
+| 1 | [Graph Change](/Announcements/Types/GraphChange) | social graph changes | no | no |
+| 2 | [Broadcast](/Announcements/Types/Broadcast) | a public post | YES | YES |
+| 3 | [Reply](/Announcements/Types/Reply) | a public response to a Broadcast | YES | YES |
+| 4 | [Reaction](/Announcements/Types/Reaction) | a public visual reply to a Broadcast | no | YES |
+| 5 | [Profile](/Announcements/Types/Profile) | a profile | YES | YES |
```

Comment:
I wonder if users should tombstone `Profile` announcements or just post new `Profile` announcements with their updated information?

Reply:
I'm strongly in favor of extremely limited application of tombstone, i.e. it's only when a signature was invalid.  For example, I don't think an update to profile information should be done as a tombstone, if that's what you mean.  If you mean instead of allowing Tombstone on Profiles, and instead having people simply update their profile, I can see the usefulness of Tombstone as a way of indicating there was a problem. It might also help with forensics / diagnostics later (but maybe that's a weak argument).

Reply:
I'm for removing Profiles from the Tombstone list.